deploy/kubernetes/examples/demo-snapshotclass.yaml

```diff
@@ -8,4 +8,4 @@ parameters:
   csi.storage.k8s.io/snapshotter-secret-name: demo-secret
   csi.storage.k8s.io/snapshotter-secret-namespace: default
   snapshot_name_prefix: demoSnapshot   # Optional.
-  pool: demo-pool                      # Mandatory only for DS8000 Family.
+  pool: demo-pool                      # Optional. Used to create the snapshot on a different pool than the source.
```

Comment:
`Used` -> `Use`.
don't forget to (ask @rivkap to) to update the example in the user guide, and to add a note that it's not supported on a9k

Reply:
done
CSI-2948